src/test/java/org/folio/innreach/repository/ItemContributionOptionsConfigurationRepositoryTest.java

```diff
@@ -0,0 +1,39 @@
+package org.folio.innreach.repository;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.test.context.jdbc.Sql;
+
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@Sql(scripts = {"classpath:db/central-server/pre-populate-central-server.sql",
+  "classpath:db/itm-contrib-opt-conf/pre-populate-itm-contrib-opt-conf.sql"})
+class ItemContributionOptionsConfigurationRepositoryTest extends BaseRepositoryTest {
```

Comment:
Test should cover all basic scenarios like create/update/delete/retrieve plus exceptional cases. For now it only verifies retrieval of entity and single exceptional case